css/style.css

```diff
@@ -0,0 +1,303 @@
+/** Global Styles **/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  background-color: #eee;
+}
+
+#pagina {
+  position: relative;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+/** Global Styles **/
+
+
+.container {
+  margin: 0 auto;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.btn {
+  height: 40px;
+  padding: 0 20px;
+  font-weight: bold;
+  font-size: 1em;
+  border-radius: 2px;
+  cursor: pointer;
+  background: transparent;
+}
+
+.row {
+  display: flex;
+}
+
+header {
+  top: 0;
+  position: fixed;
+  padding: 0px 20px;
+  min-height: 75px;
+  justify-content: space-around;
+  display: flex;
+  background-color: #eee;
+  align-items: center;
+}
+
+
+
+.logo {
```

Comment:
You may close up extra lines in your code. Among other things, that could help save some scrolling time.

Reply:
- [ ] Instead, you can write like "Kindly make sure to avoid extra empty lines to maintain readability and avoid linter errors"